regression/esbmc-solidity/uint_overflow/test.desc

```diff
@@ -0,0 +1,4 @@
+CORE
+contract.solast
+--function func --contract contract.sol
+^VERIFICATION SUCCESSFUL$
```

Comment:
Can you also add a version of this regression test that produces `VERIFICATION FAILED`?